src/_gettsim/functions/loader.py

```diff
@@ -148,13 +196,13 @@ def _load_module(path: Path, package_root: Path) -> ModuleType:
     return module
 
 
-def _convert_path_to_module_name(path: Path, package_root: Path) -> str:
+def _convert_path_to_importable_module_name(path: Path, package_root: Path) -> str:
     """
     Convert an absolute path to a Python module name.
 
     Examples
     --------
-    >>> _convert_path_to_module_name(RESOURCE_DIR / "taxes" / "functions.py")
+    >>> _convert_path_to_importable_module_name(RESOURCE_DIR / "taxes" / "functions.py")
```

Comment:
Example can't work. Also, I am not sure whether the function is strictly correct, in order to use it directly for imports it would have to return `_gettsim.taxes.functions`, right?